src/jit/emit.h

```diff
@@ -764,7 +767,7 @@ class emitter
         // x86:   24 bits
         // amd64: 16 bits
         // arm:   14 bits
-        // arm64: 14 bits
+        // arm64: 12 bits
 
         unsigned _idSmallCns : ID_BIT_SMALL_CNS;
```

Comment:
I was looking for places to free up bits.  The most obvious is the six bits used for instruction options.  I would guess these could be encoded in an enum to use less space.